src/unit_tests/syscheckd/db/test_fim_db.c

```diff
@@ -29,7 +29,7 @@
 #include "wrappers/wazuh/syscheckd/run_check_wrappers.h"
 #include "wrappers/wazuh/syscheckd/fim_diff_changes_wrappers.h"
 
-#include "db/fim_db.h"
+#include "db/db.hpp"
```

Comment:
Some places add an include to `db/db.hpp` while others add it to `db/include/db.hpp`, I believe this last one is the proper one.